src/openrct2/ride/VehicleEntry.h

```diff
@@ -15,6 +15,12 @@
 #include <array>
 #include <vector>
 
+struct SteamEffectType
```

Comment:
Not the best name, something like SteamEffectData might be a bit better, the ```Type``` part would rather apply to an enum with various steam effect types.